resources/acp-to-aep-migration/README.md

```diff
@@ -0,0 +1,164 @@
+# Migrating from ACP (Objective-C) SDK to AEP (Swift) SDK
+
+We suggest the following general process when migrating from ACP SDK to AEP SDK in a mobile app:
```

Comment:
Need to not use "we". Maybe... "The following general process is suggested when migrating..."